_compdemos/Singularity.md

```diff
@@ -1,99 +1,152 @@
 ---
 title: Using Singularity Containers
-last_modified_at: 2020-03-29
+last_modified_at: 2021-07-30
 main_author: Dirk Petersen
```

Comment:
Probably should now change main_author to someone else... whoever wants to take some ownership this round. ;)